api/src/pcapi/routes/backoffice/offer_price_limitation_rules/forms.py

```diff
@@ -0,0 +1,55 @@
+import typing
+
+from flask_wtf import FlaskForm
+import wtforms
+
+from pcapi.core.categories import categories
+from pcapi.core.categories import subcategories_v2
+
+from ..forms import fields
+from ..forms import utils
+
+
+class SearchRuleForm(FlaskForm):
+    class Meta:
+        csrf = False
+
+    category = fields.PCSelectMultipleField(
+        "Catégories", choices=utils.choices_from_enum(categories.CategoryIdLabelEnum)
+    )
+    subcategory = fields.PCSelectMultipleField(
+        "Sous-catégories",
+        choices=sorted(
+            [(s.id, s.pro_label) for s in subcategories_v2.ALL_SUBCATEGORIES if not s.is_event], key=lambda x: x[1]
+        ),
+    )
+
+
+class EditOfferPriceLimitationRuleForm(FlaskForm):
+    class Meta:
+        locales = ["fr_FR", "fr"]
+
+    rate = fields.PCDecimalField(
+        "Limite de modification du prix %",
+        use_locale=True,
+        validators=[
+            wtforms.validators.InputRequired("Information obligatoire"),
+            wtforms.validators.NumberRange(min=0, message="Doit contenir un nombre positif"),
```

Comment:
Je joue un peu avec la fonctionnalité sur ta branche de développement, et j'ai une fâcheuse exception en remplissant « 1000 » dans le pourcentage autorisé en créant (ou modifiant) une règle :
```
pc-backoffice    | sqlalchemy.exc.DataError: (psycopg2.errors.NumericValueOutOfRange) numeric field overflow
pc-backoffice    | DETAIL:  A field with precision 5, scale 4 must round to an absolute value less than 10^1.
pc-backoffice    | 
pc-backoffice    | [SQL: INSERT INTO offer_price_limitation_rule ("subcategoryId", rate) VALUES (%(subcategoryId)s, %(rate)s) RETURNING offer_price_limitation_rule.id]
pc-backoffice    | [parameters: {'subcategoryId': 'CARTE_CINE_ILLIMITE', 'rate': Decimal('10.0000')}]
```
C'est lié au format choisi en base de données, qui ne permet pas de stocker un nombre supérieur à 9,9999.
En effet, x11 sur le prix d'une offre, c'est beaucoup, mais autant protéger la saisie et la validation du formulaire pour éviter une erreur technique.

Suggestion (adapte selon ta préférence) :
```suggestion
            wtforms.validators.NumberRange(min=0, max=999, message="La limite de modification doit être un nombre entre 0 et 999 %"),
```